.github/workflows/tests.yml

```
@@ -18,3 +18,18 @@ jobs:
         with:
           command: test
           args: -q
+
+  fmt:
+    name: Format
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+      - run: rustup component add rustfmt
+      - uses: actions-rs/cargo@v1
+        with:
+          command: fmt
+          args: -all -- --check
```

This should be `--all` right?
```suggestion
          args: --all -- --check
```

---

Yep 🤦